explainer.md

```diff
@@ -132,10 +132,15 @@ enum PurchaseType {
 };
 
 dictionary ItemDetails {
-  DOMString itemId;
-  DOMString title;
+  required DOMString itemId;
+  required DOMString title;
+  required PaymentCurrencyAmount price;
   DOMString description;
-  PaymentCurrencyAmount price;
+  // Timestamps are durations in ms.
+  DOMTimeStamp SubscriptionPeriod;
```

Comment:
These periods should be DOMStrings in ISO 8601 format (https://en.wikipedia.org/wiki/ISO_8601#Durations), not DOMTimeStamp.

That's because they aren't strict durations that can be converted to seconds. It might be "P1M" which means "one month" and does not correspond to any specific number of seconds.

Reply:
Also they should all start with lowercase letters.

Reply:
Done